polysh/remote_dispatcher.py

```diff
@@ -250,7 +250,7 @@ def handle_read(self):
             return
         global nr_handle_read
         nr_handle_read += 1
-        new_data = super(RemoteDispatcher, self).handle_read()
+        new_data = self._handle_read_chunk()
```

Comment:
Wasn't the previous coding better?  I couldn't understand why we are changing it.